### PR TITLE
fix: docstring for l1 norm evaluator

### PR DESCRIPTION
**Pull request by rutujasurve94**
Docstring for `l1 norm evaluator`

### CODE REVIEW

File: evaluators/embedding/L1NormEvaluator/__init__.py [hunk 1]
```diff
@@ -7,15 +10,16 @@ class L1NormEvaluator(BaseEmbeddingEvaluator):
     """
     :class:`L1NormEvaluator` evaluates the distance between actual
     and desired embeddings computing the L1 Norm between them
+
+    :param actual: the embedding of the document
+        (resulting from an Encoder)
+    :param desired: the expected embedding of the document
+    :param args: additional positional arguments.
+    :param kwargs: additional positional arguments.
+    :return: the evaluation metric value for the request document
```
Comment (Yongxuanzhang) [inline evaluators/embedding/L1NormEvaluator/__init__.py:19]: These are the docstrings of method not init() I think. So should keep them under def evaluate()?

File: evaluators/embedding/L1NormEvaluator/README.md [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+
+Initialise L1NormEvaluator:
+
+`L1NormEvaluator(model_path='pretrained', channel_axis=1, metas=metas, model_name=MobileNetV2)`
+
+Users can use Pod images in several ways:
+
+- Run with Docker (`docker run`)
+  - ```bash
+    docker run jinahub/pod.evaluator.l1normevaluator:0.0.8-0.9.28 --port-in 55555 --port-out 55556
```
Comment (Yongxuanzhang) [inline evaluators/embedding/L1NormEvaluator/README.md:27]: Version is not accurate
```suggestion
    docker run jinahub/pod.evaluator.l1normevaluator:0.0.6-1.0.4 --port-in 55555 --port-out 55556
```

File: evaluators/embedding/L1NormEvaluator/README.md [hunk 3]
```diff
@@ -1,3 +1,53 @@
...
+  - ```bash
+    jina pod --uses hub/example/config.yml --port-in 55555 --port-out 55556
+    ```
+
+- Docker command
+
+  - Specify the image name along with the version tag. The snippet below uses Jina version `0.9.20`
+
+  - ```bash
+    docker pull jinahub/pod.evaluator.l1normevaluator:0.0.8-0.9.28
```
Comment (Yongxuanzhang) [inline evaluators/embedding/L1NormEvaluator/README.md:52]:
```suggestion
    docker pull jinahub/pod.evaluator.l1normevaluator:0.0.6-1.0.4
```

File: evaluators/embedding/L1NormEvaluator/README.md [hunk 4]
```diff
@@ -1,3 +1,53 @@
...
+    ```
+
+- Conventional local usage with `uses` argument
+  - ```bash
+    jina pod --uses hub/example/config.yml --port-in 55555 --port-out 55556
+    ```
+
+- Docker command
+
+  - Specify the image name along with the version tag. The snippet below uses Jina version `0.9.20`
```
Comment (Yongxuanzhang) [inline evaluators/embedding/L1NormEvaluator/README.md:49]:
```suggestion
  - Specify the image name along with the version tag. The snippet below uses Jina version `1.0.4`
```

File: evaluators/embedding/L1NormEvaluator/__init__.py (hunk 1 above)
Comment (rutujasurve94) [reply evaluators/embedding/L1NormEvaluator/__init__.py]: Yes, makes sense, I'm moving it back under the `evaluate` method

**Comment by rutujasurve94** [comment]
Thanks for the review, I've updated the PR as per above suggestions

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**